x-pack/plugins/security_solution/public/detections/components/host_isolation/isolate.tsx

```diff
@@ -47,7 +52,7 @@ export const IsolateHost = React.memo(
       []
     );
 
-    const caseCount: number = useMemo(() => caseIds.length, [caseIds]);
+    const caseCount: number = useMemo(() => casesInfo.length, [casesInfo]);
```

Comment:
I think that the overhead of the `useMemo` here surpass the benefits. `const caseCount: number = casesInfo.length` would not cause re-renders as the value is a primitive type.